docs/finality-provider-operation.md

```diff
@@ -589,29 +584,30 @@ this feature is implemented.
 
 ### 5.3. Jailing and Unjailing
 
-A finality provider can be jailed for the following reasons:
-1. Missing Votes
-   - Not submitting finality votes for a certain number of blocks
-   - Missing votes when the FP has positive voting power
+A finality provider can be jailed for missing votes when they have voting power. 
+However, it's important to understand that:
 
-2. Missing Public Randomness
-   - Not committing public randomness for blocks
-   - Required before voting can occur
+1. A finality provider must commit public randomness for a block before they can 
+have voting power for that block
+2. If a finality provider fails to commit public randomness for a block, they 
+will have zero voting power for that block
+3. Having zero voting power means they cannot vote (and thus cannot miss votes) 
+for that block
+4. Therefore, missing public randomness commitments indirectly prevents voting 
+but does not directly cause jailing
```

Comment:
I feel we don't need to explain this chunk in the jailing section. It is more related to `what makes a fp have voting power` and the answer is more complex than randomness. This is a big topic and we should have a page for it and we refer to that page here. This would be a TODO for me